server/initializers/migrations/0715-sponsored-videos.ts

```diff
@@ -0,0 +1,24 @@
+import * as Sequelize from 'sequelize'
+
+async function up (utils: {
+  transaction: Sequelize.Transaction
+  queryInterface: Sequelize.QueryInterface
+  sequelize: Sequelize.Sequelize
+  db: any
+}): Promise<void> {
+  const data = {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false,
+    allowNull: false
+  }
+  await utils.queryInterface.addColumn("video", "sponsored", data)
```

Comment:
We prefer to not use default values in PostgreSQL. So please do another query using `changeColumn` to remove the `defaultValue` field

Also please use `'` instead of `"`